src/CustomElements/v1/CustomElements.js

```diff
@@ -61,6 +61,10 @@ var CustomElementDefinition;
     return reservedTagList.indexOf(name) !== -1;
   }
 
+  function createTreeWalker (root) {
+    return doc.createTreeWalker(root, NodeFilter.SHOW_ELEMENT, null, false);
```

Comment:
Previously I think you had a comment about how IE needed these arguments. That was useful.